### PR TITLE
Failed: Could not create directory, file exists and is not a director…

### PR DESCRIPTION
**Pull request by helena**
5 successful builds so far for multi-jvm / multi-node  #27661. We could consider getting this merged after review / approvals but still leaving the issue open until we are sure across the several CI jobs that have failed. There are around 4.

There's a larger problem related to this that I've been looking into and chatting with Eugene about. Investigation in progress, in the background.

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,3 +1,8 @@
+libraryDependencies += Defaults.sbtPluginExtra(
+  "com.eed3si9n" % "sbt-assembly" % "0.14.10",
+  (sbtBinaryVersion in pluginCrossBuild).value,
+  (scalaBinaryVersion in pluginCrossBuild).value)
```
Comment (johanandren) [inline project/plugins.sbt]: Why all this vs `addSbtPlugin` ?

**Review by johanandren — APPROVED** [review]
LGTM but curious about the very specific way to add a plugin.

**Review by patriknw — APPROVED** [review]
LGTM, I assume this is an override of the sbt-assembly dependency

File: project/plugins.sbt (hunk 1 above)
Comment (helena) [reply project/plugins.sbt]: Overriding the version used in `sbt-multi-jvm`

Comment (johanandren) [reply project/plugins.sbt]: Ah, so it's updating a transitive dependency. Thanks for explaining!

Comment (helena) [reply project/plugins.sbt]: Now I just have to beat @scala-steward to the eventual plugin version upgrade and will remove the override in that PR.

**Pull request merged**